helix-core/src/main/java/org/apache/helix/spectator/RoutingTableProvider.java

```diff
@@ -83,17 +87,27 @@
   private ExecutorService _reportExecutor;
   private Future _reportingTask = null;
 
+  protected static final  String DEFAULT_PROPERTY = "HELIX_DEFAULT_PROPERTY";
```

Comment:
DEFAULT_PROPERTY_TYPE and DEFAULT_STATE_TYPE are cleaner, IMHO.

Reply:
Done.